src/kontrol/prove.py

```diff
@@ -978,101 +987,104 @@ def _create_cse_accounts(
     new_accounts: list[KInner] = []
     new_account_constraints: list[KApply] = []
 
-    storage_map = KVariable(contract_name + '_STORAGE', sort=KSort('Map'))
-    new_accounts.append(Foundry.symbolic_account(contract_name, contract_code, storage_map))
+    storage_map: KInner = KVariable(contract_name + '_STORAGE', sort=KSort('Map'))
+
+    singly_occupied_slots = [
+        slot for (slot, count) in Counter([field.slot for field in storage_fields]).items() if count == 1
+    ]
 
     for field in storage_fields:
+        field_name = contract_name + '_' + field.label.upper()
+        # Processing of strings
         if field.data_type == 'string':
-            lookup = KEVM.lookup(storage_map, intToken(field.slot))
-            length_byte_lt32 = ltInt(
-                KApply(
-                    '_&Int_',
-                    [
-                        intToken(127),
-                        KApply(
-                            '_>>Int_',
-                            [
-                                lookup,
-                                intToken(1),
-                            ],
-                        ),
-                    ],
-                ),
-                intToken(32),
-            )
-            length_byte_positive = geInt(
-                KApply(
-                    '_&Int_',
-                    [
-                        intToken(127),
-                        KApply(
-                            '_>>Int_',
-                            [
-                                lookup,
-                                intToken(1),
-                            ],
-                        ),
-                    ],
-                ),
-                intToken(0),
-            )
-            lowest_bit_not_set = eqInt(
-                intToken(0),
-                KApply(
-                    '_&Int_',
-                    [
-                        intToken(1),
-                        lookup,
-                    ],
-                ),
+            string_contents = KVariable(field_name + '_S_CONTENTS', sort=KSort('Bytes'))
+            string_length = KVariable(field_name + '_S_LENGTH', sort=KSort('Int'))
+            string_structure = KApply(
+                'asWord',
```

Comment:
Here you can use `as_word`, imported `from kevm_pyk.kevm`.
```suggestion
            string_structure = as_word (
```

Reply:
Yes.